### ISSUE TITLE
Include Bank doesn't include the Reagent Bank

### ISSUE THREAD

**Issue by mcmlx8** [wow-addon/FarmIt2#20]
If you're including the bank in your tracking, once items go into the Reagent Bank those items are no longer counted.

**Comment by med0rian** [comment]
I don't think this should be a big deal. If possible, I'll try to have a look tomorrow. Fortunately, the API is well documented. :)

**Comment by med0rian** [comment]
PR #22 are able to do the counting :)


And is a feature in [https://github.com/wow-addon/FarmIt2/releases/tag/v11.0.2](url)
